api/context.go

```diff
@@ -0,0 +1,30 @@
+package api
```

Comment:
is there a reason that we don't have these just in the gcontext package? And why we need to add a string to them?